crates/ruff/src/rules/flake8_use_pathlib/mod.rs

```diff
@@ -67,7 +69,7 @@ mod tests {
         let snapshot = format!("{}_{}", rule_code.noqa_code(), path.to_string_lossy());
         let diagnostics = test_path(
             Path::new("flake8_use_pathlib").join(path).as_path(),
-            &settings::Settings::for_rule(rule_code),
+            &settings::Settings::for_rule(rule_code).with_target_version(PythonVersion::latest()),
```

Comment:
@zanieb - We should probably make this the default for `Settings::for_rule`, what do you think? (But we can hash out in separate PR, I'm just trying to get tests passing here.)